onboarding/src/Components/utils.js

```diff
@@ -41,3 +41,8 @@ export function getCookie(name) {
 	return cookieValue;
 }
 
+export function validEmail(email){
+	let regExp = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+	return regExp.test(String(email));
+}
+
```

Comment:
On page: http://html5pattern.com/Emails is this tip: 
![image](https://user-images.githubusercontent.com/51723246/97438327-eebfdb00-1924-11eb-823a-a4b5dda51106.png)

Reply:
Great :-)
and yet pity I didn't get this message during evaluation :/

I had strange experience that no e-mail values were accepted on that input O.o
That's why I did this regexp :/
Source:
https://stackoverflow.com/questions/46155/how-to-validate-an-email-address-in-javascript

Reply:
You mean input had no validation? I think it is because input is not in form or you not handle it with `onSubmit()` event?